### ISSUE TITLE
Support unicode U+2044

### ISSUE THREAD

**Issue by j4-m** [jakeboone02/numeric-quantity#20]
https://www.compart.com/en/unicode/U+2044

Presently, `1⁄2 teaspoon of ground cumin` yields:

```
[
  {
    "quantity": 1,
    "quantity2": null,
    "unitOfMeasure": null,
    "description": "⁄2 teaspoon of ground cumin",
    "isGroupHeader": false
  }
]
```

But `1/2 teaspoon of ground cumin` (U+002F)  yields:
```
[
  {
    "quantity": 0.5,
    "quantity2": null,
    "unitOfMeasure": "teaspoon",
    "description": "of ground cumin",
    "isGroupHeader": false
  }
]
```

I think this library should either support both characters or translate to the expected format however the former is more conventional when expressing fractions.

**Comment by jakeboone02** [comment]
Hey, @j4-m!  Thanks for the feature request.  This has been implemented and deployed in [v1.0.2](https://github.com/jakeboone02/numeric-quantity/releases/tag/v1.0.2).

**Comment by j4-m** [comment]
Amazing @jakeboone02 thanks for looking at this so quickly.
